network-api/networkapi/mozfest/templates/partials/mozfest_footer.html

```diff
@@ -0,0 +1,43 @@
+{% with wrapper_class="py-5" %}
```

Comment:
While it looks like it isn't, this footer _does_ get used, the React code for the footer just kicks in and replaces its content.

In order to give us some control, I've assigned the `<body>` a class which for mozfest pages always contains `mozfest`, so we can probably update our React code for the footer in `main.js` to do something different for mozfest pages, so they look the way they're supposed to.

Reply:
> While it looks like it isn't, this footer does get used, the React code for the footer just kicks in and replaces its content.

How is the React code kick in and replace this footer content?

Reply:
We're loading the same `main.js` that the foundation site does, so it sees the `join-us` class and goes "aha! I can drop a `<JoinUs>` component in there~"

Reply:
👏 👏 👏 ✨ ✨